agent/consul/config.go

```diff
@@ -522,11 +522,13 @@ func DefaultConfig() *Config {
 		TombstoneTTLGranularity:              30 * time.Second,
 		SessionTTLMin:                        10 * time.Second,
 		ACLTokenMinExpirationTTL:             1 * time.Minute,
-		ACLTokenMaxExpirationTTL:             24 * time.Hour,
+		// Duration is stored as an int64. Setting the default max
+		// to the max possible duration (approx 290 years).
+		ACLTokenMaxExpirationTTL: 1<<63 - 1,
```

Comment:
@johnlanda : Do we need to update any docs associated with this parameter?

Strangely, it seems like the relevant docs I could find never mention the current 24 hour limit ... so maybe there is nothing to update?
- https://developer.hashicorp.com/consul/commands/acl/token/create#expires-ttl
- https://developer.hashicorp.com/consul/commands/acl/auth-method/create#max-token-ttl

Reply:
I could not find any mention of the limit either. I will take another look though to be sure.

Reply:
I do not believe there is anything to update. It looks like that was an undocumented limit.